test/column/colSpan.test.ts

```diff
@@ -34,7 +34,7 @@ describe('colSpan', () => {
         }
       });
     }
-    setup({ columns, rows, summaryRows: [1, 2] });
+    setup({ columns, rows, bottomSummaryRows: [1, 2] });
```

Comment:
Should we test colSpan with top summary rows as well?

Reply:
Added some tests